app/views/pictures/_stats.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <!-- Comments -->
     <div id="" class="comment-stat stats col s3">
-      12 Comments
+      <%= picture.comments.count %> Comments
```

Comment:
Pluralize this bitch! "1 comments" is not correct!

https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize